code/src/components/app-table/components/app-table-sort-label/AppTableSortLabel.types.ts

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from "react";
+
+import { TableSortLabelProps } from "@mui/material/TableSortLabel";
+
+
+export type AppTableSortLabelProps = TableSortLabelProps & {
+  sortDirection: 'asc' | 'desc';
```

Comment:
Create common `type SortOrder = 'asc' | 'desc'` and reuse it

Reply:
Added